marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/jobs.sjs

```diff
@@ -296,3 +296,16 @@ module.exports.updateJob = module.amp(
   }
 );
 
+module.exports.deleteJobDocs = module.amp(
```

Comment:
Let's just call this "deleteJobs" - technically, it's deleting Job and Batch docs, so "JobDocs" is a little misleading because it makes the reader think it only deletes Job docs. Also, "deleteJobs" is reasonable because both Job and Batch docs are in a collection named "Jobs".